data/human/pirate jobs.txt

```diff
@@ -3711,3 +3711,720 @@ mission "Republic Assassination [2]"
+	on complete
+		"pirate jobs" ++
+		payment 3000
+		dialog "Before you can even speak, the group hands you <payment> and vanishes into the <planet> crowd."
+
+
+
+mission "Pirate bulk transport to <planet> [0]"
+	name "Bulk transport to <planet>"
+	job
+	repeat
+	description "This large group of <bunks> anarchists is hoping to find work on <destination>, and they will pay you <payment> to take them there."
+	passengers 11 30 .8
+	to offer
+		random < 300
+		and
+			"reputation: Pirate" > -10
+	source
+		government "Pirate"
+	destination
+		distance 2 12
+		government "Pirate"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on fail
+		"reputation: Pirate" -= 2
+	on complete
+		"pirate jobs" ++
+		payment 1000
+		dialog "You wish the anarchists the best of luck on <planet>, and collect your payment of <payment>."
+
+mission "Pirate bulk transport to <planet> [1]"
+	name "Bulk transport to <planet>"
+	job
+	repeat
+	description "This large group of <bunks> anarchists are looking to escape local trouble by moving to <destination>, and they will pay you <payment> to take them there."
+	passengers 11 30 .8
+	to offer
+		random < 300
+		and
+			"reputation: Pirate" > -10
+	source
+		government "Pirate"
+	destination
+		distance 2 12
+		government "Pirate" "Republic" "Deep" "Free Worlds" "Syndicate"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on fail
+		"reputation: Pirate" -= 2
+	on complete
+		"pirate jobs" ++
+		payment 1000
+		dialog "You wish the anarchists the best of luck on <planet>, and collect your payment of <payment>."
+
+mission "Pirate bulk transport to <planet> [2]"
+	name "Bulk transport to <planet>"
+	job
+	repeat
+	description "This large group of <bunks> anarchists are hoping to start new lives on <destination>, and they will pay you <payment> to take them there."
+	passengers 11 30 .8
+	to offer
+		random < 300
+		and
+			"reputation: Pirate" > -10
+	source
+		government "Pirate"
+	destination
+		distance 2 12
+		government "Pirate"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on fail
+		"reputation: Pirate" -= 2
+	on complete
+		"pirate jobs" ++
+		payment 1000
+		dialog "You wish the anarchists the best of luck on <planet>, and collect your payment of <payment>."
+
+mission "Pirate bulk transport to <planet> [3]"
+	name "Bulk transport to <planet>"
+	job
+	repeat
+	description "A large ship scrapping operation is looking to move a team of <bunks> employees to their facility on <destination>, and they will pay you <payment> to take them there."
+	passengers 11 30 .6
+	to offer
+		random < 350
+		and
+			"reputation: Pirate" > -10
+	source
+		government "Pirate"
+	destination
+		distance 1 4
+		government "Pirate"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on fail
+		"reputation: Pirate" -= 2
+	on complete
+		"pirate jobs" ++
+		payment 1250
+		dialog "You find your contact on <planet>, and after a lengthy head count, they hand you your payment of <payment>."
```

Comment:
```suggestion
		dialog "You find your contact on <planet> and, after a lengthy head count, they hand you your payment of <payment>."
```

You could also probably use both commas or just remove the last one, but I think this is the most grammatically correct way to do it.